packages/legend-application-studio/src/stores/sidebar-state/ProjectOverviewState.ts

```diff
@@ -183,6 +197,78 @@ export class ProjectOverviewState {
     }
   }
 
+  *fetchDependants(): GeneratorFn<void> {
+    const groupId =
+      this.editorStore.projectConfigurationEditorState.projectConfiguration
+        ?.groupId;
+    const artifactId =
+      this.editorStore.projectConfigurationEditorState.projectConfiguration
+        ?.artifactId;
+    const version = this.editorStore.sdlcState.projectVersions[0]?.id.id;
```

Comment:
should we take the latest version for this?

Reply:
Yea, latest released versions for now in this one since user can choose version in the extension and once we allow for current workspace we can change?